modelling/src/test/java/org/axonframework/modelling/repository/AccessSerializingRepositoryTest.java

```diff
@@ -171,4 +174,14 @@ public String applyStateChange(UnaryOperator<String> change) {
             throw new UnsupportedOperationException("Not implemented");
         }
     }
+
+    @Nonnull
+    private static UnitOfWork aUnitOfWork() {
```

Comment:
Nit: Why have custom static methods if you can use the versions on the `UnitOfWorkTestUtils` directly?